src/components/input/input.stories.mdx

```diff
@@ -33,3 +41,64 @@ You can also add `c-input` to `<select>` elements, in which case you’ll get a
     })}
   </Story>
 </Preview>
+
+## Elastic textarea
+
+Add `js-elastic-textarea` to a `<textarea>` element\* to have it expand as the user types more than will fit.
+
+If the `rows` attribute is set, it will be used as the minimum height.
+
+<p>
+  <small>* see also the JavaScript usage notes below</small>
+</p>
+
+<Preview>
+  <Story name="Elastic Textarea" height="350px">
+    {() => {
+      // Use storybook hooks to trigger JS after story renders
+      // @see https://github.com/storybookjs/storybook/issues/7786
+      useEffect(() => {
```

Comment:
I read through the issue and I'm still confused about this? Is this different from a react hook? What exactly is it doing?

Reply:
@calebeby My understanding is this is a React hook, under the hood. But I don't know enough about Storybook to confirm this.

Reply:
What does it do that is different from the react useEffect (i.e. why do we import this from storybook, instead of react)?

Reply:
¯\\\_(ツ)_/¯

Reply:
@calebeby because when I googled "how do I get my JS to work after a story renders in storybook" I ended up on that issue I linked in the comment which had this code sample. I don't actually understand React hooks (they came out after I finished my React training) so I'm open to being told we don't need the Storybook hook exposing this, but I'd need some help with what we _should_ use instead.

Reply:
No worries, I'm fine with leaving this as-is, I don't have a suggestion. I was just trying to understand how this is working